src/backend/joanie/tests/core/utils/test_contract_definition_generate_document_context.py

```diff
@@ -554,3 +545,136 @@ def test_utils_contract_definition_generate_document_context_course_data_section
         self.assertIsInstance(contract.context["course"]["price"], str)
         self.assertEqual(order.total, Decimal("999.99"))
         self.assertEqual(contract.context["course"]["price"], "999.99")
+
+    @override_settings(
+        JOANIE_DOCUMENT_ISSUER_CONTEXT_PROCESSORS={
+            "contract_definition": [
+                "joanie.tests.core.utils.test_contract_definition_generate_document_context._processor_for_test_suite"  # pylint: disable=line-too-long
+            ]
+        }
+    )
+    @mock.patch(
+        "joanie.tests.core.utils.test_contract_definition_generate_document_context._processor_for_test_suite",  # pylint: disable=line-too-long
+        side_effect=_processor_for_test_suite,
+    )
```

Comment:
By adding a constant below 
```
def _processor_for_test_suite(context):
     […]

PROCESSOR_PATH = "joanie.tests.core.utils.test_contract_definition_generate_document_context._processor_for_test_suite"
```

We can simplify this part:
```suggestion
    @override_settings(
        JOANIE_DOCUMENT_ISSUER_CONTEXT_PROCESSORS={
            "contract_definition": [PROCESSOR_PATH]
        }
    )
    @mock.patch(PROCESSOR_PATH, side_effect=_processor_for_test_suite)
```